src/main/java/uk/gov/hmcts/ccd/endpoint/ui/QueryEndpoint.java

```diff
@@ -106,6 +99,21 @@ public QueryEndpoint(
             .orElseThrow(() -> new ResourceNotFoundException("No case types found")));
     }
 
+    @RequestMapping(value = "/caseworkers/{uid}/jurisdictions", method = RequestMethod.GET)
+    @ApiOperation(value = "Get jurisdictions available to the user")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "List of jurisdictions for the given access criteria"),
+        @ApiResponse(code = 404, message = "No jurisdictions found for given access criteria")})
+    public List<JurisdictionDisplayProperties> getJurisdictions(@RequestParam(value = "access") String access) {
+        List<JurisdictionDisplayProperties> jurisdictions = Arrays.asList(
+            getUserProfileOperation.execute(accessMap.get(access)).getJurisdictions());
```

Comment:
Missing validation of the `access` property?

Reply:
added validation